src/Libraries/PythonNodeModelsWpf/IronPythonCompletionProvider.cs

```diff
@@ -596,124 +979,249 @@ protected Type TryGetType(string name)
                 var val = matches[i].Groups[6].Value.Trim();
                 paramMatches.Add(name, val);
             }
+
             return paramMatches;
+        }
+
+        public List<string> findClrReferences(string code)
```

Comment:
needs a summary tag

Reply:
this along with one other method above can actually be private, will replace them now